### ISSUE TITLE
moving cursor disfluency

### ISSUE THREAD

**Issue by AELe** [VSCodeVim/Vim#4255]
**Describe the bug**
Running vs code some time, the plug-in vim moving cursor(up down left right) disfluency.

**To Reproduce**

1.startup vs code
2.edit code some time

**Expected behavior**
Moving cursor fluency

**Screenshots**

**Environment (please complete the following information):**
Mint linux 4.15.0-66-generic

- Extension (VsCodeVim) version:
- VSCode version:1.9.0
- OS:

**Additional context**

**Comment by J-Fields** [comment]
Could you please clarify? You mean when you hold `j`, for instance, the cursor stutters? Does this happen on every file or just large ones? Do you have any related remappings?

**Comment by AELe** [comment]
Hi @J-Fields 
I opened a folder approximately 4.2GB.When I running VS code for a long time half a day or a day, when I hold the keyboard j,k,h,l, It begins to stutter.Yes it happens every file in loaded folder.I didn`t remap.If I close the VS code and restart it the phenomenon will disappear.

**Comment by alexdreptu** [comment]
I'm experiencing something similar but when holding `C-e` and `C-y`. I'm not sure if it has the same cause. The scroll is not fluid but rather choppy.

**Comment by salembeats** [comment]
@alexdreptu Yeah, I actually moved to WebStorm because of this. Simple things like moving one character in a given direction became really choppy on a 2018 MBP that had been really snappy in the past.

**Comment by alexdreptu** [comment]
@salembeats vscode scrolling and overall UI is very smooth normally. The issue started after I updated both vscode and the vim extension. If I use `j`, `k` or the arrow keys it scrolls as it normally does, although it uses a bit too much CPU. Luckily I don't need to use `C-e`, `C-y` or `j` and `k` too much, I scroll with `C-d` and `C-u` and jump around the lines where I need to edit. But it's still an inconvenience.
